test/models/transfer_test.rb

```diff
@@ -15,12 +15,12 @@
 class TransferTest < ActiveSupport::TestCase
   setup do
     @transfer = transfers(:valid)
-    file = Rails.root.join('test', 'fixtures', 'files', 'a_pdf.pdf')
-    @transfer.files.attach(io: File.open(file), filename: 'a_pdf.pdf')
+    # file = Rails.root.join('test', 'fixtures', 'files', 'a_pdf.pdf')
+    # @transfer.files.attach(io: File.open(file), filename: 'a_pdf.pdf')
```

Comment:
Non-blocking, but I'd vote for removing these commented-out lines unless there's a reason we might return to attaching files in the setup..